test/tests/functional/pbs_multi_sched.py

```diff
@@ -130,6 +130,11 @@ def get_tzid(self):
             tzone = 'America/Los_Angeles'
         return tzone
 
+    def set_scheduling(self, scheds=[], op=False):
```

Comment:
Don't default to [].  This will create a single list which will be used for all function calls.  You don't modify it, but if you had, the changes would persist from call to call.  If you want an empty list, you set the default to None, and then check for it and set scheds to [] in the function itself.

An option is to set scheds to self.server.scheds.  That way the default is all the schedulers.